agent/bench-scripts/pbench-run-benchmark-sample

```diff
@@ -164,12 +171,27 @@ if (! $pp_only) {
                     $cmd = eval "\$_='$cmd'; $r; return scalar \$_";
                 }
             }
-            my $result = ssh_hosts(\@{ $hosts{$type} }, $cmd, $sample_dir . "-" . $type, $base_dir);
+            if ($type eq "server") {
+                # Servers need to be backgrounded so we can proceed to running
+                # the clients.
+                $server_ssh_logdir = ssh_hostdirs(\@{ $hosts{$type} }, $cmd, $base_dir, 1);
+            } else {
+                ssh_hostdirs(\@{ $hosts{$type} }, $cmd, $base_dir, 0);
+            }
+            if ($type eq 'server') {
+                my $seconds = scalar @{ $hosts{$type} };
+                printf "Waiting %d seconds for servers to be ready\n", $seconds;
+                sleep($seconds);
```

Comment:
This seems really fragile.  Can we use a redis server here to coordinate servers and clients a little cleaner?

Reply:
I suppose one could add a more robust mechanism, but I think that is an enhancement for a later PR.